manticore/core/cpu/x86.py

```diff
@@ -5620,6 +5633,102 @@ def VZEROUPPER(cpu):
 ################################################################################
 #Calling conventions
 
+class ABI:
```

Comment:
it looks like the ABI abstraction refactor was undone here, and things are being added back?